### ISSUE TITLE
Sizeof lacks support for width 3 vectors

### ISSUE THREAD

**Issue by TPolzer** [gpu/JOCL#13]
3 element vectors are required since OpenCL 1.1.

**Comment by gpu** [comment]
That's right. Maybe I did not add them because their size is defined to be the same as for the 4-element vectors (c.f. https://github.com/KhronosGroup/OpenCL-Headers/blob/ad081fe10bce51091360e5225aff933703451dd6/cl_platform.h#L1280 ).  But I'll add them nevertheless, for clarity.
